authz_analyzer/datastores/aws/aws_ptrp_package/aws_ptrp/services/s3/s3_actions.py

```diff
@@ -89,6 +142,33 @@ def difference(self, other: 'ResolvedActionsSingleStmt'):
 
             self.resolved_stmt_actions.difference_update(actions_to_difference)
 
+    def is_not_resource_annotated(self) -> bool:
+        return self._not_resource_annotated
+
+    @staticmethod
```

Comment:
this is not the right place, maybe move it to ResolvedS3BucketActions as a load function 
Look at the return value

Reply:
change to 'load....' (cls function)